examples/plane/limiters_advection.jl

```diff
@@ -211,49 +215,56 @@ for (k, ne) in enumerate(ne_seq)
         end
 
         # Compute hyperviscosity for the tracer equation by splitting it in two diffusion calls
+        ystar = similar(y)
         @. ystar.ρq = wdiv(grad(y.ρq / y.ρ))
-        Spaces.weighted_dss!(ystar)
+        Spaces.weighted_dss!(ystar.ρq)
         @. ystar.ρq = -D₄ * wdiv(y.ρ * grad(ystar.ρq))
 
         # Add advective flux divergence
-        @. ystar.ρ = -wdiv(y.ρ * u)         # contintuity equation
-        @. ystar.ρq += -wdiv(y.ρq * u)      # adevtion of tracers equation
-    end
+        @. dy.ρ = beta * dy.ρ - alpha * wdiv(y.ρ * u)         # contintuity equation
+        @. dy.ρq = beta * dy.ρq - alpha * wdiv(y.ρq * u) + alpha * ystar.ρq   # advection of tracers equation
+
 
-    function stage_callback!(ydoublestar, integrator, parameters, t)
-        space = parameters.space
         min_Q = parameters.min_Q
         max_Q = parameters.max_Q
 
         if lim_flag
             # Call quasimonotone limiter, to find optimal ρq (where ρq gets updated in place)
             Limiters.quasimonotone_limiter!(
-                ydoublestar.ρq,
-                ydoublestar.ρ,
+                dy.ρq,
+                dy.ρ,
```

Comment:
Answer to self, for future reference. No, we're not applying it to the tendency, we're still applying it to the state. The way it's written in CliMATimeSteppers interface it's just like a temporary increment, since we use `alpha = dt` and `beta = 1`.